agent/provider/src/rules.rs

```diff
@@ -371,13 +368,9 @@ impl RulesManager {
         }
     }
 
-    fn verify_permissions(&self, cert: &str, demand_permissions_present: bool) -> Result<()> {
-        let mut required = vec![CertPermissions::OutboundManifest];
-
-        if demand_permissions_present {
-            required.push(CertPermissions::UnverifiedPermissionsChain);
-        }
-
+    //TODO delete when deleting X509 permissions feature
```

Comment:
This comment is misleading, because ultimately we will have two use-cases for audited-payload rule:
1. Incoming is x509 cert: in this case we don't check permissions as there are no permissions in cert
2. Incoming is golem cert: in this case we need to check cert permissions for it